pkg/controller/vitessshardreplication/reconcile_drain.go

```diff
@@ -118,7 +118,7 @@ func (r *ReconcileVitessShard) reconcileDrain(ctx context.Context, vts *planetsc
 	}
 
 	// Get all the tablet records for the shard.
-	tablets, err := wr.TopoServer().GetTabletMapForShard(ctx, keyspaceName, vts.Spec.Name)
+	tablets, err := wr.TopoServer().GetTabletMapForShardByCell(ctx, keyspaceName, vts.Spec.Name, vts.Spec.GetCells().UnsortedList())
```

Comment:
Edit the comment above this line:

```go
// Get all the tablet records for the shard, in cells to which we deploy.
// We ignore tablets in cells we don't deploy, since we assume there's
// a separate operator instance handling drains on those tablets.
```

Reply:
Done.